test/jdk/java/awt/print/PrinterJob/SwingUIText.java

```diff
@@ -90,20 +118,9 @@ public static void createUI() {
         frame.getContentPane().add(panel);
         frame.pack();
         frame.setVisible(true);
-
-        PrinterJob job = PrinterJob.getPrinterJob();
-        PageFormat pf = job.defaultPage();
-        job.setPrintable(new SwingUIText(), pf);
-        if (job.printDialog()) {
-            try { job.print(); }
-            catch (Exception e) {
-              e.printStackTrace();
-              throw new RuntimeException(e);
-            }
-        }
+        return frame;
     }
 
-
     static void displayText(JPanel p, String text) {
         JPanel panel = new JPanel();
         panel.setLayout(new GridLayout(2,1));
```

Comment:
```suggestion
        panel.setLayout(new GridLayout(2, 1));
```

There is also another purely syntax issue slightly below this by adding spaces around the `+`:
```
JButton button = new JButton("Print " + text);
```